src/frontend/components/TourGuide/TourGuide.scss

```diff
@@ -0,0 +1,77 @@
+.introjs-tooltip {
+  background: #222530;
+  color: var(--text-default);
+  border-radius: var(--space-xs);
+  box-shadow: 0px 4px 20px rgba(0, 0, 0, 0.25);
+  border: 1px solid var(--accent);
+  min-width: 320px;
+}
+
+.introjs-tooltiptext {
+  font-size: var(--text-xl);
+  line-height: 1.5;
+  padding: var(--space-sm);
+}
+
+.introjs-button {
+  border: 2px solid var(--stroke-brand-strong, #ab82fe);
```

Comment:
Why do we have a hexacolor here?

Reply:
when using css variables the second value is a backup. This came from figma.